product_docs/docs/epas/16/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx

```diff
@@ -319,6 +422,71 @@ The `FOPEN` function opens a file for I/O.
 
  Variable of type `FILE_TYPE` containing the file handle of the opened file.
 
+## FOPEN_NCHAR
+
+The `FOPEN_NCHAR` procedure opens file `filename` in the directory identified by `location` to read and/or write Unicode character set data. 
+
+```sql
+FOPEN_NCHAR(location VARCHAR2, filename VARCHAR2, open_mode VARCHAR2, max_linesize INTEGER)
+```
+
+### Parameters 
+
+`location`
+
+ Directory name of the directory containing the file to open, as stored in `pg_catalog.edb_dir.dirname`.
```

Comment:
```suggestion
 The Directory name containing the file to open is stored in `pg_catalog.edb_dir.dirname`.
```

Reply:
Name of the directory containing the file to open, stored in `pg_catalog.edb_dir.dirname`.

Not sure about this...the original is confusing. Does this mean, The directory name you specify is stored in the `pg_catalog.edb_dir.dirname`. ?? If so, I think it'll be less confusing to use multiple sentences.

I'm not even sure about the first part of the sentence. Maybe it'd be clearer as: 

Name of the directory that contains the file you want to open.